package.json

```diff
@@ -30,90 +30,89 @@
     "postpack": "rimraf addon-test-support/**/*.js",
     "clean": "git clean -x -f",
     "docs": "documentation build --document-exported \"addon-test-support/@ember/test-helpers/index.js\" --config documentation.yml --markdown-toc-max-depth 3 -f md -o API.md",
-    "lint": "npm-run-all lint:*",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
     "lint:js": "eslint --cache .",
     "lint:ts": "tsc -p tsconfig.json --noEmit",
     "postpublish": "npm-run-all clean",
     "release": "release-it",
     "start": "ember serve",
-    "test": "ember test",
+    "test": "concurrently \"npm:test:*\" --names \"test:\"",
     "test:all": "ember try:each"
   },
   "peerDependencies": {
-    "ember-source": ">=3.8.0"
+    "ember-source": "^3.28.0 || ^4.0.0"
   },
   "dependencies": {
     "@ember/test-waiters": "^3.0.0",
-    "@embroider/macros": "^1.6.0",
-    "@embroider/util": "^1.6.0",
+    "@embroider/macros": "^1.10.0",
+    "@embroider/util": "^1.9.0",
     "broccoli-debug": "^0.6.5",
     "broccoli-funnel": "^3.0.8",
-    "ember-cli-babel": "^7.26.6",
-    "ember-cli-htmlbars": "^5.7.1",
-    "ember-destroyable-polyfill": "^2.0.3"
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-htmlbars": "^6.1.1"
   },
   "devDependencies": {
     "@babel/cli": "^7.18.9",
+    "@babel/core": "^7.20.5",
     "@babel/preset-typescript": "^7.18.6",
     "@ember/optional-features": "^2.0.0",
-    "@glimmer/component": "^1.0.4",
+    "@glimmer/component": "^1.1.2",
     "@glimmer/interfaces": "^0.84.1",
     "@glimmer/reference": "^0.84.1",
     "@tsconfig/ember": "^1.0.1",
     "@types/ember": "~4.0.0",
+    "@types/ember-testing-helpers": "^0.0.4",
```

Comment:
We should *not* need this. I'll look at where/how we're using it below, but since `ember-testing-helpers` is shipped by Ember itself, we *should* be able to catch those in our preview and stable types (…yet another back-port! Wheeee!). It's not a blocker, though: we can use this for now since it has no direct dependencies on Ember.

Reply:
I didn't see it used anywhere. ember-cli-update just re-sorted the dependencies so it shows as green here

Reply:
w/o it I get lots of `error TS2304: Cannot find name 'jQuery'.` but no other errors. It's all re: this code:

https://github.com/emberjs/ember-test-helpers/blob/412a4ba23bd800abeaca763248341d8e94fff679/addon-test-support/%40ember/test-helpers/settled.ts#L42-L66

Reply:
Oh, I see. Odd! Very bad. Also, we should, uhh… remove that code as part of our breaking change release. 😂 

Let's leave it as is for now, but tackle it when you do the further push for using the preview/stable types.